wordpress-mobile/WordPress-FluxC-Android
# [Hazmat Shipping] Add hazmat category parameter support

:warning: Depends on https://github.com/woocommerce/woocommerce-android/pull/9601

Summary
==========
Partially fix issue [#9575](https://github.com/woocommerce/woocommerce-android/issues/9595) by introducing the Hazmat parameter support to the Repository and Store layers of the Shipping Label classes.

How to Test
==========
1. Ensure the code is good and CI and unit tests are green. No testable behavior was introduced here.

Update release notes:
